packages/sass-node-modules-importer/source/resolvers/directory.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+import { log } from '../logger'
+import { Resolver } from '../types'
+
+export const resolveFromPackageJson: Resolver = (options, directory) => {
+	log(options, 'attempting to resolve package.json')
+
+	const packageJsonPath = path.resolve(directory, 'package.json')
+
+	if (!fs.existsSync(packageJsonPath)) return null
+
+	log(options, 'found package.json')
+	const packageJson = JSON.parse(fs.readFileSync(packageJsonPath).toString())
+
+	if (!Object.prototype.hasOwnProperty.call(packageJson, 'style')) {
+		log(options, 'no style specified in package.json, aborting')
+		return null
+	}
+
+	log(options, 'found style')
+	return path.resolve(directory, packageJson.style)
```

Comment:
unless this returns null by default

Reply:
this will be faulty

Reply:
This is intentional. I added a JSDoc comment to explain why above.